de-lib/src/main/java/org/iplantc/de/teams/client/TeamsView.java

```diff
@@ -113,6 +113,14 @@
         String searchFieldEmptyText();
 
         String teamSearchFailed();
+
+        String leaveTeamSuccess(Group group);
+
+        String leaveTeamHeader(Group group);
```

Comment:
Maybe `team` instead of `group` for this method and the `leaveTeamSuccess` method.